### PR TITLE
chore: copy element finders & browser from protractor

### PR DESCRIPTION
**Pull request by tsaleksandrova**

### CODE REVIEW

File: src/browser/browser.js [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+var webdriver_js_extender = require('webdriver-js-extender');
+var Command = require('selenium-webdriver/lib/command').Command;
+var CommandName = require('selenium-webdriver/lib/command').Name;
+
+// modules copied to UIVeri5
+var element = require('../element/element');
+var locators = require('../element/locators');
+
+// TODO module to copy from Protractor
+var protrDebugger = require('protractor/built/debugger');
```
Comment (maximnaidenov) [inline src/browser/browser.js:11]: as discussed, we don't need this debugger

File: src/browser/browser.js [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+/**
+ * The same as {@code webdriver.WebDriver.prototype.executeAsyncScript} but with extra logging
+ *
+ * @private
+ * @param {string} scriptName A description for debugging purposes.
+ * @param {...*} scriptArgs The arguments to pass to the script.
+ * @returns {!webdriver.promise.Promise.<T>} A promise that will resolve to the scripts return value.
+ * @template T
+ */
+ProtractorBrowser.prototype._executeAsyncScript = function (scriptName, ...scriptArgs) {
```
Comment (maximnaidenov) [inline src/browser/browser.js:108]: we already have something similar in uiveri5, maybe we can use it directly ?

File: src/browser/browser.js [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+/**
+ * If set to false, Protractor will not wait for UI5
+ * tasks to complete before interacting with the browser. This can cause
+ * flaky tests, but should be used if, for instance, your app continuously
+ * polls an API with $timeout.
+ *
+ * Call waitForAngularEnabled() without passing a value to read the current
+ * state without changing it.
+ */
+ProtractorBrowser.prototype.waitForAngularEnabled = function (enabled) {
```
Comment (maximnaidenov) [inline src/browser/browser.js:147]: ignoreSynchronization is not used in uiveri5 => remove it

File: src/browser/browser.js [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+      return selenium_webdriver.promise.controlFlow().execute(function () {
+        return protrDebugger;
+      }, 'add breakpoint to control flow');
+    });
+};
+
+/**
+ * See browser.explore().
+ */
+ProtractorBrowser.prototype.enterRepl = function (opt_debugPort) {
```
Comment (maximnaidenov) [inline src/browser/browser.js:315]: same as debugger, we don't need this repl and explorer

File: src/element/locators.js [hunk 5]
```diff
@@ -0,0 +1,198 @@
+/**
```
Comment (maximnaidenov) [inline src/element/locators.js:1]: as discussed, we don't need any of those protractor custom locators

**Comment by tsaleksandrova** [comment]
moving to new branch - noptor

**Pull request closed without merging**